pyproject.toml

```diff
@@ -26,47 +26,45 @@ dependencies = [
     "wandb>=0.16.3",
     "protobuf>=3.20.2",
     "urllib3>=1.26.18,<2",
-    "pydantic>=2.6.4",
-    "pydantic-yaml>=1.2.0",
```

Comment:
I kept only the == as there was redundancy here and having both would have resulted in having ==, but am happy to put >= if the intention was to keep them updated

Reply:
yeah, i changed it back to make sure it worked via pip-compile.